content/Day-1/_index.md

```diff
@@ -1,43 +1,38 @@
 
 We can use labeling to help us find the feeling(s) that we may need to address or feelings/verbiage we may need to avoid. Addressing or avoiding the feeling will be determined by the situation, so use your best judgment. We use labeling and summarizing to better understand how the speaker is feeling and to ensure we have the right emotion before we try to help. When we have the right emotion labeled, we can attempt a solution, and gauge if the speaker is still feeling the same emotion or if they have moved towards a positive or less negative position.  Learning to label is a valuable skill that can be used in a variety of situations. By learning this skill you can help to prevent conflict and problem escalation. 
 
-## Validation Through Normalization 
+### Validation Through Normalization 
 
 The experience the customer is having is real to them so the more we demonstrate understanding, the quicker we can resolve the problem with the customer. To normalize a customer experience, we can tell them we would feel the same or similar if we were in the situation. However, we want to avoid saying “I know how you feel” this can create a new problem. They may feel the need to own the feeling they are having at the moment. Also, we do not know about all the other things that are going on in their life. 
 
 We can echo back to the speaker/customer the feeling we think they’re experiencing. For example, “it sounds like you &lt;restate the problem]> and you are feeling &lt;labeled feeling>.” At this point, we pause and wait for acknowledgement to confirm the accuracy of the restatement and labeling . Once they have acknowledged the feeling we can say we understand why they are feeling the feeling they have and try to take it a step further by inviting them to help solve the problem. Remember, whatever the customer is feeling is not our fault but what they are feeling is real to them so focus on handling the situation to the best of your ability.By normalizing and validating someone’s emotional experiences we can help them to feel less alone and more understood. When you hit a roadblock in your troubleshooting steps,  escalate the problem to the appropriate manager or supervisor. 
 
-## Technical Communication Analyst 
-
-### **Primary Task - Identify the Problem** 
+### 3 Technical Communication Task 
 
+**Primary Task - Identify the Problem** 
```

Comment:
The tasks for this section may break the page up a bit as ### headers

Reply:
not sure what this means can you elaborate on this issue?